include/llbuild/BuildSystem/BuildNode.h

```diff
@@ -84,6 +84,12 @@ class BuildNode : public Node {
     return exclusionPatterns;
   }
 
+  std::vector<StringRef> mustScanAfterPaths;
```

Comment:
This declaration should move up above with the other ivars so it doesn't have public visibility.